src/parser/statement.js

```diff
@@ -1069,27 +1129,52 @@ export default class StatementParser extends ExpressionParser {
       // `get\n*` is an uninitialized property named 'get' followed by a generator.
       // a getter or setter
       method.kind = key.name;
-      this.parsePropertyName(method);
-      if (this.isNonstaticConstructor(method)) {
-        this.raise(method.key.start, "Constructor can't have get/set modifier");
+      if (this.match(tt.hash)) {
+        // private getter/setter
+        this.expectPlugin("classPrivateMethods");
+        // The private # wouldn't have been eaten as the "async" was in front of it.
```

Comment:
Comment is incorrect.

Reply:
Huh? Maybe my wording was confusing.

What I'm trying to say is that it's now being eaten, but it wasn't detected as being "private" in the code above (`let isPrivate = ...`). Is that wrong?

Reply:
The "async" part is incorrect. We're in the "get"/"set" path here.